app/controllers/clients_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
   def show
     @identities = @client.identities
     @movements = @client.movements
+    @country_of_birth = @client.identities.first.country_of_birth.name
```

Comment:
i don't think this is right.
A client can have many identities. We don't want to show only one of them

A client can have multiple countries of birth, because country names and borders change - and so does their info.
The country of birth needs to be shown for that identity -- of which there can be many